frontend/src/app/data-entry/agent-form/agent-historical-person-form/agent-historical-person-form.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, DestroyRef, OnDestroy, OnInit } from "@angular/core";
+import { FormService } from "../../shared/form.service";
+import {
+    DataEntryAgentHistoricalPersonGQL,
+    DataEntryHistoricalPersonsGQL,
+    DataEntryUpdateAgentGQL,
+    DataEntryUpdateAgentMutation,
+} from "generated/graphql";
+import {
+    combineLatest,
+    debounceTime,
+    filter,
+    map,
+    Observable,
+    share,
+    switchMap,
+    withLatestFrom,
+} from "rxjs";
+import { FormControl, FormGroup } from "@angular/forms";
+import { formStatusSubject } from "../../shared/utils";
+import { takeUntilDestroyed } from "@angular/core/rxjs-interop";
+import { MultiselectOption } from "../../shared/multiselect/multiselect.component";
+import { MutationResult } from "apollo-angular";
+import { ToastService } from "@services/toast.service";
+
+interface HistoricalPerson {
+    describes: string[];
+}
+
+type HistoricalPersonForm = {
+    [key in keyof HistoricalPerson]: FormControl<string[]>;
+};
+
+@Component({
+    selector: "lc-agent-historical-person-form",
+    templateUrl: "./agent-historical-person-form.component.html",
+    styleUrls: ["./agent-historical-person-form.component.scss"],
+})
+export class AgentHistoricalPersonFormComponent implements OnInit, OnDestroy {
+    private id$ = this.formService.id$;
+
+    private agent$ = this.id$.pipe(
+        switchMap((id) => this.agentQuery.watch({ id }).valueChanges),
+        map((result) => result.data.agentDescription),
+        share()
+    );
+
+    private allHistoricalPersons$ = this.historicalPersonsQuery
+        .fetch()
+        .pipe(map((result) => result.data.historicalPersons));
+
+    private historicalPersonOptions$: Observable<MultiselectOption[]> =
+        this.allHistoricalPersons$.pipe(
+            map((persons) =>
+                persons.map((person) => ({
+                    value: person.id,
+                    label: `${person.name} (${
+                        person.dateOfBirth?.displayDate ?? " ?"
+                    } – ${person.dateOfDeath?.displayDate ?? "? "})`,
```

Comment:
If the dates are not filled in, you'll get, for example, `Radegund (? - ?)`. Looks a bit odd; at first I thought this was an emoticon of some sort.

I don't think we'll have that many historical figures anyway, so it would be fine to just use the name here.